src/main/java/uk/gov/hmcts/reform/ccd/document/am/controller/WelcomeController.java

```diff
@@ -1,18 +1,20 @@
 package uk.gov.hmcts.reform.ccd.document.am.controller;
 
-import static org.springframework.http.ResponseEntity.ok;
-
-import java.util.UUID;
-
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.http.ResponseEntity;
+import org.springframework.http.converter.HttpMessageConversionException;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.document.am.controller.advice.exception.*;
```

Comment:
Kindly disable your intellij import optimise and remove the * based imported classes.

Reply:
Done